src/HDC1080.c

```diff
@@ -48,7 +50,14 @@ float GetTemperature(void)
 	uint8_t regaddr = HDC1080_TEMP_REG;
```

Comment:
This would be a good simple check for errors, I agree. However, it's not very robust. What happens if the I2C line gets held high? Then we're in real trouble... The only way to fix these real errors is to clock through the problem. I agree we should add robust error checking to our application code, but lets get bare bone code that works for now, and add error correction later.